src/Main.elm

```diff
@@ -89,10 +89,11 @@ type LoadingMsg
     | CvHentet (Result Http.Error Cv)
     | CvOpprettet (Result Http.Error Cv)
     | RegistreringsProgresjonHentet (Result Http.Error RegistreringsProgresjon)
+    | IkkeGjørNoe
 
 
 type SuccessMsg
-    = BrukerSierHeiIIntroduksjonen
+    = BrukerSierHeiIIntroduksjonen MeldingsLogg
```

Comment:
Her syns jeg ikke at meldingsloggen burde ligge i Msg-en, for den ligger jo allerede i SamtaleSeksjon. I `update` kan du heller gjøre en `case ... of` på samtale, for å få tilgang til meldingsloggen der.